x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.js

```diff
@@ -4,48 +4,74 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { KIBANA_STATS_TYPE_MONITORING } from '../../../common/constants';
+import {
+  LOGGING_TAG,
+  KIBANA_MONITORING_LOGGING_TAG,
+  KIBANA_STATS_TYPE_MONITORING
+} from '../../../common/constants';
 import { opsBuffer } from './ops_buffer';
+import Oppsy from 'oppsy';
+
+const LOGGING_TAGS = [LOGGING_TAG, KIBANA_MONITORING_LOGGING_TAG];
+
+
+class OpsMonitor {
+  constructor(server, buffer, interval) {
+    this._buffer = buffer;
+    this._interval = interval;
+    this._oppsy = new Oppsy(server);
+    this._server = server;
+  }
+
+  start = () => {
+    this._oppsy.on('ops', (event) => {
+      // Oppsy used to provide this, but doesn't anymore. Grab it ourselves.
+      this._server.listener.getConnections((_, count) => {
+        event.concurrent_connections = count;
+        this._buffer.push(event);
+      });
+    });
+
+    this._oppsy.on('error', console.log);
+    this._oppsy.start(this._interval);
+  };
+
+  stop = () => {
+    this._oppsy.stop();
+    this._oppsy.removeAllListeners();
+  };
+}
+
 
 /*
  * Initialize a collector for Kibana Ops Stats
  */
-export function getOpsStatsCollector(server) {
-  let monitor;
+export function getOpsStatsCollector(server, kbnServer) {
   const buffer = opsBuffer(server);
-  const onOps = event => buffer.push(event);
-
-  const start = () => {
-    monitor = server.plugins['even-better'].monitor;
-    monitor.on('ops', onOps);
-  };
-  const stop = () => {
-    if (monitor) {
-      monitor.removeListener('ops', onOps);
-    }
-  };
+  const interval = kbnServer.config.get('ops.interval');
```

Comment:
Just want to double check with @tsullivan here.

We are currently already using this same config value to configure how often we fetch ops data from hapi right? 

[Here](https://github.com/elastic/kibana/blob/master/src/server/logging/configuration.js#L55) from [here]()https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.js#L19) from [here](https://github.com/elastic/kibana/blob/master/src/server/kbn_server.js#L182)

Reply:
This config value is supposed to define the granularity of the ops event emitter, which means how often the event fires, and how large the window is for including stats. It looks like this PR isn't changing that